snmp/datadog_checks/snmp/data/profiles/f5-big-ip.yaml

```diff
@@ -117,6 +117,8 @@ metrics:
         name: sysMultiHostCpuSoftirq
       - OID: 1.3.6.1.4.1.3375.2.1.7.5.2.1.10
         name: sysMultiHostCpuIowait
+      - OID: 1.3.6.1.4.1.3375.2.1.7.5.2.1.11
+        name: sysMultiHostCpuUsageRatio
```

Comment:
Looking closer at this metric, I think `sysMultiHostCpuUsageRatio`, I think the `forced_type: percent` might not be suitable. It seems different to other metrics like `sysMultiHostCpuUser`, `sysMultiHostCpuNice`, etc in this table, that are monotonically increasing.

```
1.3.6.1.4.1.3375.2.1.7.5.2.1.5 | sysMultiHostCpuNice | 0 | 0 | The time spent by the specified processor running niced processes for the associated host.
-- | -- | -- | -- | --
1.3.6.1.4.1.3375.2.1.7.5.2.1.6 | sysMultiHostCpuSystem | 0 | 0 | The time spent by the specified processor servicing system calls for the associated host.
1.3.6.1.4.1.3375.2.1.7.5.2.1.7 | sysMultiHostCpuIdle | 0 | 0 | The time spent by the specified processor doing nothing for the associated host.
1.3.6.1.4.1.3375.2.1.7.5.2.1.8 | sysMultiHostCpuIrq | 0 | 0 | The time spent by the specified processor servicing hardware interrupts for the associated host.
1.3.6.1.4.1.3375.2.1.7.5.2.1.9 | sysMultiHostCpuSoftirq | 0 | 0 | The time spent by the specified processor servicing soft interrupts for the associated host.
1.3.6.1.4.1.3375.2.1.7.5.2.1.10 | sysMultiHostCpuIowait | 0 | 0 | The time spent by the specified processor waiting for external I/O to complete for the associated host.
```

Do you know what the raw value look like ? From the snmpwalk here, it looks like the value is a ratio x100 https://github.com/DataDog/integrations-core/blob/ba807d4f848b98eaaf6e77c31ab09a79fb961ccf/snmp/tests/compose/data/f5-big-ip.snmprec#L4167-L4168

```
1.3.6.1.4.1.3375.2.1.7.5.2.1.11.1.48.1|66|5  => 5%
1.3.6.1.4.1.3375.2.1.7.5.2.1.11.1.48.2|66|7  => 7%
```

So, I think it might be more suitable to add a different section like below. That will send metrics with type gauge instead of `forced_type: percent`:

```
  - MIB: F5-BIGIP-SYSTEM-MIB
    table:
      OID: 1.3.6.1.4.1.3375.2.1.7.5.2
      name: sysMultiHostCpuTable
    symbols:
      - OID: 1.3.6.1.4.1.3375.2.1.7.5.2.1.11
        name: sysMultiHostCpuUsageRatio
    metric_tags:
      - column:
          OID: 1.3.6.1.4.1.3375.2.1.7.5.2.1.3
          name: sysMultiHostCpuId
        tag: cpu
```

Reply:
Good call, thanks for spotting this

Reply:
Added `sysMultiHostCpuUsageRatio` as a different section. https://github.com/DataDog/integrations-core/pull/10924/commits/29be2df08acbd72be6a3428d9f972d32adfaabdf